mantidproject/mantid
# Added unit tests for ReflDataProcessorPresenter

After the creation of the `ReflDataProcessorPresenter` in #18227, no unit tests had been added to check its behaviour. This PR fixes this by adding unit tests relevant to this class.

**To test:**

- Ensure the new tests pass.
- Ensure that the new tests actually work as described.
- Identify any missing areas that could use unit testing not covered by the new tests or by `GenericDataProcessorPresenterTest.h`.

Fixes #18498.

*Does not need to be in the release notes.*

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
